src/plugins/navigation/review/navigator.js

```diff
@@ -552,6 +559,14 @@ function navigatorFactory(config, map, context) {
         navigator.trigger('jump', position);
     }
 
+    if (config["reviewLayout"] === 'fizzy') {
+        navigatorTpl = fizzyTpl;
+        navigatorTreeTpl = fizzyTreeTpl;
+        // hack to not allow activate/deactivate bookmarking on icon click in item button
```

Comment:
Just a question (you don't have to change anythning): so ideally you should have passed `isFizzyLayout` condition through to js methods below, but since you don't want to make many changes, you use this workaround. Right?

Reply:
Yes) All this task about workarounds and compromises.